bcda/responseutils/writer.go

```diff
@@ -304,7 +305,7 @@ func WriteBundleResponse(bundle *fhirmodels.Bundle, w http.ResponseWriter) {
 		return
 	}
 
-	w.Header().Set("Content-Type", "application/json")
+	w.Header().Set(constants.ContentType, constants.ContentType)
```

Comment:
maybe this one should be w.Header().Set(constants.ContentType, constants.JsonContentType) for consistency? 
JsonContentType as 2nd parameter instead of ContentType duplicated